src/Nodes/ModuleItemNode.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { ModuleItem } from "../Model/ModuleItem";
+import { INode } from "./INode";
+
+export class ModuleItemNode implements INode {
+    constructor(public readonly moduleItem: ModuleItem) {
```

Comment:
public?

Reply:
This property should be public because we need to use it in other place